change/@microsoft-teams-js-f4f055db-c85c-4ea2-b470-278759a1aafb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "In `dialog.ts` moved `Resize` functionality in a subcapability named `update`.",
```

Comment:
Another change that should be included is that the input type is now different. The parameter has changed from type DialogInfo to DialogSize.